pkg/kubelet/stats/cri_stats_provider_windows.go

```diff
@@ -0,0 +1,89 @@
+// +build windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package stats
+
+import (
+	"time"
+
+	"github.com/Microsoft/hcsshim"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	statsapi "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
+)
+
+// listContainerNetworkStats returns the network stats of all the running containers.
+func (p *criStatsProvider) listContainerNetworkStats() (map[string]*statsapi.NetworkStats, error) {
+	containers, err := hcsshim.GetContainers(hcsshim.ComputeSystemQuery{})
```

Comment:
I believe hcsshim.GetContainers(hcsshim.ComputeSystemQuery{}) returns the virtual machine as well if you don't specify the Types filter. To get only container list you need to call this api with Container filter.
hcsshim.GetContainers(hcsshim.ComputeSystemQuery{Types: []string{"Container"}})

Reply:
Thanks for tip, fixed in the new commit.